fastdl.py

```diff
@@ -55,9 +56,23 @@ def filesEqual(rootfile, fdfile, bz2format = False):
 					print("File {} compared for {:.2f} seconds".format(rootfile, time() - prevtime))
 					return True
 
+def initBlacklist(path_to_blacklist):
+	BlackListedFiles = []
+	if not os.path.exists(path_to_blacklist):
+		print("BlackList not found at {}. ignoring...".format(path_to_blacklist))
+	else:
+		print("BlackList found at {}. Parsing files...".format(path_to_blacklist))
+		blacklist = open(path_to_blacklist, "r")

```

Comment:
Also would like to ask you to change ``open()`` to use ``with`` statement instead, to follow the same style through out the code.